src/assisted_test_infra/test_infra/helper_classes/config/controller_config.py

```diff
@@ -10,6 +10,9 @@
 
 @dataclass
 class BaseNodeConfig(BaseConfig, ABC):
+    # CI build id
+    build_id: str = None
```

Comment:
From what I'm seeing we don't need this variable and if we do, BaseNodeConfig is definitely not where it should be.
BuildID is specific for the CI and it's more of a global variable.

Reply:
do you think that the controller should have access to the global parameters? It should have access to the controller config